docs/API.md

```diff
@@ -0,0 +1,26 @@
+# API
+
+## Socket events
+
+### Client
+
+Client-side API sends socket events to the server.
+
+sendToServer.js handles events dispatching.
```

Comment:
these sentences are the beginning of the introduction of the overall doc, not just this section.